reference-stores/nuxt/components/search/SearchInput.vue

```diff
@@ -16,24 +17,14 @@
             pointer-events-none
           "
         >
-          <svg
-            class="h-5 w-5 text-gray-400"
-            xmlns="http://www.w3.org/2000/svg"
-            viewBox="0 0 20 20"
-            fill="currentColor"
-            aria-hidden="true"
-          >
-            <path
-              fill-rule="evenodd"
-              d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z"
-              clip-rule="evenodd"
-            />
-          </svg>
+          <!-- eslint-disable vue/no-v-html -->
+          <span class="h-5 w-5 text-gray-400" v-html="searchIcon" />
+          <!-- eslint-enable vue/no-v-html -->
         </div>
         <input
-          :id="`search-${_uid}`"
+          id="search-header"
```

Comment:
This is resulting in 2 elements on at least the home page with the id `search-header`. 
Looks like 1 in the header and 1 in the search modal. Should probably make them unique or allow a prop to set them in context. 


Error output from Accessibility insights for the web: 


> Title: WCAG 4.1.1: Ensures every id attribute value of active elements is unique (.pl-10.pr-3.leading-5)
> Tags: Accessibility, WCAG 4.1.1, duplicate-id-active
> 
> Issue: Ensures every id attribute value of active elements is unique (duplicate-id-active - https://accessibilityinsights.io/info-examples/web/duplicate-id-active)
> 
> Target application: Nuxt Reference Store - https://nacelle-nuxt-reference-store-git-ls-1558-nuxt-re-5d6b80-nacelle.vercel.app/
> 
> Element path: `.ml-auto > .px-2.lg\:ml-6.lg\:justify-end > .max-w-lg.lg\:max-w-xs.w-full > .relative > .pl-10.pr-3.leading-5`
> 
> Snippet: `<input id="search-header" name="search-header" placeholder="Search" type="search" class=" block w-full pl-10 pr-3 py-2 border border-gray-300 rounded-md leading-5 bg-white placeholder-gray-500 focus:outline-none focus:placeholder-gray-400 focus:ring-1 focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm ">`
> 
> How to fix: 
> Fix any of the following:
>   Document has active elements with the same id attribute: search-header
> 
> Environment: Microsoft Edge version 101.0.1210.32
> 
> ====
> 
> This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.